src/a4/test/BoardSpaceTest.java

```diff
@@ -0,0 +1,94 @@
+package a4.test;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import a4.domain.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class BoardSpaceTest {
+	BoardSpaceFactory test_factory = new BoardSpaceFactory();
+	BoardSpace test_space = test_factory.getBoardSpace("Open");
+	ArrayList<Player> test_players = new ArrayList<Player>();
+	Player player1 = new Player();
+	Player player2 = new Player();
+	Player player3 = new Player();
+	Player player4 = new Player();
+
```

Comment:
The variables still need to be global, but we can initialize everything in the initialize if you want. It's basically the same either way since players do not get used except to add to an array List.